lib/profile_events_queue_handler.ts

```diff
@@ -26,8 +26,8 @@ const DEFAULT_REQUEST_TIMEOUT_MS = 10000;
 const isProduction = process.env.NODE_ENV === "production";
 
 // Needed to call notifications API
-const notificationApiUrl = getRequiredStringEnv("ADMIN_API_URL");
-const adminApiKey = getRequiredStringEnv("ADMIN_API_KEY");
+const notificationApiUrl = getRequiredStringEnv("NOTIFICATION_API_URL");
```

Comment:
we use messages in the API nomenclature, I would either use `MESSAGES_API_URL` or `PUBLIC_API_URL`